personalized_active_learning/experiments/active_learning/wiki/aggression.py

```diff
@@ -0,0 +1,135 @@
+import active_learning.algorithms as algorithms
+from personalized_active_learning.active_learning_flows import (
+    StandardActiveLearningFlow,
+)
+from personalized_active_learning.datamodules import AggressionDataModule
+from personalized_active_learning.datamodules.base import SplitMode
+from personalized_active_learning.embeddings import EmbeddingsCreator
+from personalized_active_learning.metrics.personal_metrics import (
+    PersonalizedMetricsCallback,
+)
+from personalized_active_learning.models import Baseline
+from personalized_nlp.utils import seed_everything
+from personalized_nlp.utils.experiments import product_kwargs
+from settings import DATA_DIR
+
+# False positive https://github.com/Lightning-AI/lightning/issues/11856
+warnings.filterwarnings("ignore", category=PossibleUserWarning)
+os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+os.environ["WANDB_START_METHOD"] = "thread"
+
+
+if __name__ == "__main__":
+    wandb_project_name = "PNW_AL_Wiki_Aggression"
+    wandb_entity_name = "be-active"  # None if you don't want to use entity
+    datamodule_cls = AggressionDataModule
+    model_cls = Baseline
+    use_cuda = False
+    embeddings_dir = DATA_DIR / "wiki_data" / "embeddings" / "aggression"
+    activelearning_kwargs_list = product_kwargs(
+        {
+            "text_selector_cls": [
+                algorithms.RandomSelector,
+                #algorithms.BalancedConfidenceSelector,
+                #algorithms.BalancedClassesPerUserSelector,
+                #algorithms.BalancedClassesPerTextSelector,
+                # algorithms.ConfidenceSelector,
+                # algorithms.MaxPositiveClassSelector,
+                # algorithms.ConfidenceAllDimsSelector,
+                # algorithms.Confidencev2Selector,
+                # algorithms.RandomImprovedSelector,
+            ],
+            "max_amount": [100_000],
+            "step_size": [None],
+            "amount_per_user": [2],
+            "stratify_by_user": [True],
+        }
+    )
+    datamodule_kwargs_list = product_kwargs(
+        {
+            "embeddings_creator": [
+                EmbeddingsCreator(
+                    directory=embeddings_dir,
+                    embeddings_type="labse",
+                    use_cuda=use_cuda,
+                ),
+            ],
+            "past_annotations_limit": [None],
+            "split_mode": [SplitMode.TEXTS],
+            "folds_num": [10],
+            "subset_ratio": [0.2],
+            "batch_size": [32],
+            "test_fold_index": [0],  # This does cross-validation
+            "use_finetuned_embeddings": [False],
+            "major_voting": [False],
+            "min_annotations_per_user_in_fold": [20],
+        }
+    )
+    model_kwargs_list = product_kwargs({})
+    trainer_kwargs_list = product_kwargs(
+        {
+            "epochs": [20],
+            "lr": [0.0001],
+            "use_cuda": [use_cuda],
+            "monitor_metric": ["valid_loss"],
+            "monitor_mode": ["min"],
```

Comment:
[Question] Is it the best configuration of params? Or it's just copy paste? (random)

Reply:
The second:P

Reply:
I.e. copy paste